compiler/luci-interpreter/src/kernels/L2Normalize.test.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
```

Comment:
If we used the same number as TFLite for testing, it would be safe to add TensorFlow copyright.

@lemmaa , Do we have to add the phrase for TensorFlow copyright when using the same test dataset as TFLite?

Reply:
@jinevening , I'm not sure, but I think it would be nice to specify that. To be precise, the copyright displayed in the corresponding source code should be copied as it is. Would you point-out the original source code for reference?

Reply:
> Would you point-out the original source code for reference?

The test data is the same with [this test](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/l2norm_test.cc#L71-L78) in TFLite.

@struss , Can you add the below phrase at line 3?
`Copyright 2017 The TensorFlow Authors. All Rights Reserved.`

Reply:
@jinevening Okay, I will add that on all test cases that referring from tensorflow.